lib/vanagon/platform.rb

```diff
@@ -60,6 +60,9 @@ class Platform
     # Stores the local path where retrieved artifacts will be saved.
     attr_accessor :output_dir
 
+    # Stores the local path where source artifacts will be saved.
+    attr_accessor :source_dir
```

Comment:
Hmm.... I kinda think this is a little bit confusing, too easy to mix up with build sources. maybe source_output_dir? or something a bit more descriptive?